prow/plugins/trigger/trigger.go

```diff
@@ -155,10 +155,14 @@ func handlePush(pc plugins.Agent, pe github.PushEvent) error {
 func TrustedUser(ghc trustedUserClient, onlyOrgMembers bool, trustedOrg, user, org, repo string) (bool, error) {
 	// First check if user is a collaborator, assuming this is allowed
 	if !onlyOrgMembers {
-		if ok, err := ghc.IsCollaborator(org, repo, user); err != nil {
+		if users, err := ghc.ListCollaborators(org, repo); err != nil {
```

Comment:
Can we not use this terrible notation of using `else` after error checks? We return in the `if` branch, we do not need to make the `else` branch